### PR TITLE
fix: chatgo extension - Worker terminated due to reaching memory limit: JS heap out of memory

### PR DESCRIPTION
**Pull request by chuyun**
## Description

fix issues #7453 

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by mil3na — CHANGES_REQUESTED** [review]
Hi! Thank you so much for your contribution to Raycast!
My name is Milena and I will be reviewing your extension update today :)

Your changes seem to fix the issue linked, but I still get the following error as soon as I enter the `Ask Question` command. Can you please take a look?
CleanShot 2023-07-14 at 15 58 01@2x

**Comment by mil3na** [comment]
Hi! Any updates on my comment? Anything I can help you with?

**Comment by chuyun** [comment]
> Hi! Any updates on my comment? Anything I can help you with?你好！我的评论有更新吗？有什么我可以帮你的吗？

ok,I will fix it soon thanks

File: extensions/chatgo/src/hooks/useQuestion.tsx [hunk 1]
```diff
@@ -26,11 +29,11 @@ export function useQuestion(props: { initialQuestion: string; disableAutoLoad?:
             });
           }
         } catch (error) {
-          await showToast({
-            style: Toast.Style.Failure,
-            title: "Selected text couldn't load",
-            message: String(error),
-          });
+          // await showToast({
```
Comment (mil3na) [inline extensions/chatgo/src/hooks/useQuestion.tsx:32]: why is this commented? Shouldn't it really show an error?
It works now on my side, but I'm worried that an error can occur and no feedback is given to the user.

Comment (chuyun) [reply extensions/chatgo/src/hooks/useQuestion.tsx:32]: This error captures the error of getting the selected text API. It is reasonable to capture the error here but not display it on the UI

Comment (mil3na) [reply extensions/chatgo/src/hooks/useQuestion.tsx:32]: That's alright, but I'd recommend removing the commented code and documenting why you're not treating the error.

Comment (chuyun) [reply extensions/chatgo/src/hooks/useQuestion.tsx]: > That's alright, but I'd recommend removing the commented code and documenting why you're not treating the error.

ok~

**Comment by mil3na** [comment]
You need to run prettier (or `ray lint --fix` in order to make it pass the tests :)

**Pull request merged**